src/ui/components/cards/cardcomponent.js

```diff
@@ -111,6 +111,10 @@ export default class CardComponent extends Component {
 
     // Use the cardType as component name if it is not a built-in type
     const cardComponentName = cardTypes[cardType] || cardType;
+
+    // additional field(s) for dataMappings config option
+    this.result._raw.distance = this.result.distance;
```

Comment:
would it make sense to add this to answers-core's ResultFactory.ts?

Reply:
this seems more of a SDK specific implementation for user customization purposes using dataMappings, so I think maybe we shouldn't modify answers-core's result format?

Reply:
ok! should we do this in the constructor instead then?

Reply:
oh yeah, make sense. Updated!